### PR TITLE
FISH-6410: updating to jakarta 10 compatibility

### PR DESCRIPTION
**Pull request by breakponchito**
**Description**
Samples for Java EE 8 were updated to be compatible with jakarta EE 10
**Tested on local environment:**

[INFO] Reactor Summary for Java EE 8 Samples: root 1.0-SNAPSHOT:
[INFO]
[INFO] Java EE 8 Samples: root ............................ SUCCESS [  2.024 s]
[INFO] Java EE 8 Samples: test-utils ...................... SUCCESS [  1.287 s]
[INFO] Java EE 8 Samples: Servlet ......................... SUCCESS [  0.012 s]
[INFO] Java EE 8 Samples: Servlet - http2 ................. SUCCESS [  5.401 s]
[INFO] Java EE 8 Samples: Servlet - mapping ............... SUCCESS [ 11.176 s]
[INFO] Java EE 8 Samples: JAX-RS .......................... SUCCESS [  0.007 s]
[INFO] Java EE 8 Samples: JAX-RS - SSE Producer ........... SUCCESS [ 13.829 s]
[INFO] Java EE 8 Samples: Security ........................ SUCCESS [  0.007 s]
[INFO] Java EE 8 Samples: Security - Dynamic Remember-me .. SUCCESS [  9.747 s]
[INFO] Java EE 8 Samples: Security - JWT .................. SUCCESS [  9.986 s]
[INFO] Java EE 8 Samples: JSF ............................. SUCCESS [  0.016 s]
[INFO] Java EE 8 Samples: JSF - Hello World ............... SUCCESS [  9.721 s]
[INFO] Java EE 8 Samples: JSF - Extensionless Mapping ..... SUCCESS [  9.958 s]
[INFO] Java EE 8 Samples: CDI ............................. SUCCESS [  0.007 s]
[INFO] Java EE 8 Samples: CDI - Dynamic Bean .............. SUCCESS [  8.849 s]
[INFO] Java EE 8 Samples: CDI - Events Async .............. SUCCESS [ 17.271 s]
[INFO] Java EE 8 Samples: CDI - Events Priority ........... SUCCESS [  9.705 s]
[INFO] Java EE 8 Samples: CDI - Interception factory ...... SUCCESS [  8.992 s]
[INFO] Java EE 8 Samples: CDI - Qualified Lookup .......... SUCCESS [  9.545 s]
[INFO] Java EE 8 Samples: CDI - Discovery Modes ........... SUCCESS [ 31.282 s]
[INFO] Java EE 8 Samples: JPA ............................. SUCCESS [  0.005 s]
[INFO] Java EE 8 Samples: JPA - Dynamic Transaction ....... SUCCESS [  9.606 s]
[INFO] Java EE 8 Samples: JPA - Stream .................... SUCCESS [  8.916 s]
[INFO] Java EE 8 Samples: JSON-B .......................... SUCCESS [  0.007 s]
[INFO] Java EE 8 Samples: JSON-B - Mapping ................ SUCCESS [ 10.318 s]
[INFO] Java EE 8 Samples: Bean Validation 2 ............... SUCCESS [  0.006 s]
[INFO] Java EE 8 Samples: Bean Validation 2: Support for java.time API SUCCESS [  9.181 s]
[INFO] Java EE 8 Samples: JSON-P .......................... SUCCESS [  0.004 s]
[INFO] Java EE 8 Samples: JSON-P - Patch .................. SUCCESS [  8.125 s]
[INFO] Java EE 8 Samples: JSON-P - Pointer ................ SUCCESS [  8.532 s]
[INFO] Java EE 8 Samples: JSON-P - Merge .................. SUCCESS [  8.297 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  03:42 min
[INFO] Finished at: 2022-09-03T18:29:15-05:00
[INFO] ------------------------------------------------------------------------

**System environment**
Window 11, Azul JDK 11, Maven 3.8.3

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,959 +1,948 @@
...
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+
+        <repository>
+            <id>staged-jakarta-releases</id>
+            <url>https://jakarta.oss.sonatype.org/content/groups/staging/</url>
+        </repository>
```
Comment (Pandrex247) [inline pom.xml:113]: Missing the following repository for patched Payara artefacts (e.g. Jersey 3.1.0-M7.payara-p1):

```suggestion
        </repository>
        <repository>
            <id>payara-nexus-artifacts</id>
            <url>https://nexus.payara.fish/repository/payara-artifacts</url>
        </repository>
```

**Review by Pandrex247 — CHANGES_REQUESTED** [review]
Missing the following repository for patched Payara artefacts (e.g. Jersey 3.1.0-M7.payara-p1):

```
<repository>
    <id>payara-nexus-artifacts</id>
    <url>https://nexus.payara.fish/repository/payara-artifacts</url>
</repository>
```

**Comment by Pandrex247** [comment]
Could you give a quick summary of the changes to the HTTP2 test?
You seem to have rewritten it? We also appear to be quite different from how the test looks upstream?

**Comment by Pandrex247** [comment]
Other than that, after adding `payara-nexus-artifacts` repo so it can resolve Jersey - seems to work 👍

**Comment by breakponchito** [comment]
> 

Yes I basically removed the Jetty Classes that caused issues when trying to call the endpoints with http2 protocol to the default version from the JDK from the java.net.http package, also some methods for the pre and post execution that now are not needed with the new client implementation

**Review by Pandrex247 — APPROVED** [review]

**Pull request merged**